public/app/features/explore/state/query.ts

```diff
@@ -806,14 +808,14 @@ export const runLoadMoreLogsQueries = createAsyncThunk<void, RunLoadMoreLogsQuer
   }
 );
 
-const groupDataQueries = async (datasources: DataQuery[], scopedVars: ScopedVars) => {
-  const nonMixedDataSources = datasources.filter((t) => {
+const groupDataQueries = async (queries: DataQuery[], scopedVars: ScopedVars) => {
+  const nonMixedDataSourceQueries = queries.filter((t) => {
     return t.datasource?.uid !== MIXED_DATASOURCE_NAME;
   });
-  const sets: { [key: string]: DataQuery[] } = groupBy(nonMixedDataSources, 'datasource.uid');
+  const queriesByDataSource: { [key: string]: DataQuery[] } = groupBy(nonMixedDataSourceQueries, 'datasource.uid');
 
   return await Promise.all(
-    Object.values(sets).map(async (targets) => {
+    Object.values(queriesByDataSource).map(async (targets) => {
```

Comment:
just renamed those vars for cosmetics. I was confused reading `datasources: DataQuery[]`